library/neptune-triton/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "neptune-triton"
 description = "GPU implementation of neptune-compatible Poseidon hashing."
-version = "2.0.1"
+version = "2.0.0"
```

Comment:
probably should release a 2.0.2 with this, unless you yanked 2.0.1

Reply:
I did yank it. This will be updated with codegen changes applied to previous generated code before next release.

Reply:
sgtm